cmd/container-manager/manage_ctr_mgr.sh

```diff
@@ -109,9 +109,16 @@ install_docker(){
 			repo_url="https://download.docker.com/linux/fedora/docker-ce.repo"
 			sudo -E dnf -y install dnf-plugins-core
 			sudo -E dnf config-manager --add-repo "$repo_url"
-			sudo -E dnf makecache
-			docker_version_full=$(dnf --showduplicate list "$pkg_name" | grep "$docker_version" | awk '{print $2}' | tail -1)
-			sudo -E dnf -y install "${pkg_name}-${docker_version_full}"
+			if [ "$VERSION_ID" -ge "30" ]; then
+				sudo sed -i 's/$releasever/28/' /etc/yum.repos.d/docker-ce.repo
```

Comment:
Could you add a log call referencing #1954 so that anyone reading the logs can see what is happening for F30+ (and maybe spot if we can remove this code in future).

Reply:
@jodh-intel changes applied

Reply:
Hi @GabyCT - thanks, but could you make this an actual call to `warning()` so that the comment details get logged by the CI scripts?